corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -418,6 +420,13 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         templateContext: function () {
             var paginateItems = paginateOptions(this.options.currentPage, this.options.pageCount);
             var casesPerPage = parseInt($.cookie("cases-per-page-limit")) || 10;
+
+            $(function ()  {
+                var goButton = $("#pagination-go-button");
+                if (goButton.length) {
+                    kissmetrics.track.event("Accessibility Tracking - Pagination Page Loaded");
+                }
+            });
```

Comment:
This works as a check for the go button. You will get an event metric for each time a page loads with that button, which may be fine as long as the person doing the tracking is made aware. I was going for unique occurrences of pagination, but that may not be necessary or easily done.

Reply:
It looks like it'll track every time the view is rendered - which is probably what we want, just want to make sure that I understand and that it's well-documented for product. It looks like:
* User loads case list - fire `Pagination Page Loaded`
* User click next - fire `Pagination Interaction` and then fire `Pagination Page Loaded` (as page 2 is loaded)

Is that right?

Reply:
Yes, that's right, and I think is what we want.